### PR TITLE
EI-1068-phase2-apply-content-changes

### PR DESCRIPTION
**Pull request by Stephek**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Web/ViewModels/Home/HomeViewModel.cs [hunk 1]
```diff
@@ -1,20 +1,57 @@
...
+            ManageApprenticeshipSiteUrl = manageApprenticeshipSiteUrl;
+            if (!manageApprenticeshipSiteUrl.EndsWith("/"))
+            {
+                ManageApprenticeshipSiteUrl += "/";
+            }
+            AccountsAgreementsUrl = $"{manageApprenticeshipSiteUrl}accounts/{AccountId}/teams";
+            BankDetailsRequired = MapBankDetailsRequired(vrfCaseStatus, vrfVendorId);
         }
 
+        // TODO: move to api
```
Comment (David-B-Read) [inline src/SFA.DAS.EmployerIncentives.Web/ViewModels/Home/HomeViewModel.cs:43]: Could bank details required be a property of the LegalEntity?

Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Web/ViewModels/Home/HomeViewModel.cs:43]: Yes it should/could be.  Is a refactor task though.

Comment (MichaelYoung1981) [reply src/SFA.DAS.EmployerIncentives.Web/ViewModels/Home/HomeViewModel.cs:43]: Have we got a ticket for it?

Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Web/ViewModels/Home/HomeViewModel.cs]: I've moved it to the model to encapsulate it better.  Will raise a ticket to get the api to calculate and return it.

**Review by David-B-Read — APPROVED** [review]

File: src/SFA.DAS.EmployerIncentives.Web/ViewModels/Home/HomeViewModel.cs (hunk 1 above)
Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Web/ViewModels/Home/HomeViewModel.cs]: https://skillsfundingagency.atlassian.net/browse/EI-1179

**Pull request merged**